modules/runtime/nuxeo-runtime-stream/src/main/java/org/nuxeo/runtime/stream/StreamServiceImpl.java

```diff
@@ -73,6 +73,13 @@
 
     protected final Map<String, StreamProcessor> processors = new HashMap<>();
 
+    /**
+     * @since 11.2
+     */
+    public static final String STREAM_PROCESSING_DISABLE = "nuxeo.stream.processing.disable";
```

Comment:
`disable` -> `disabled`. But we have only one example of system prop about disabling: `nuxeo.vcs.fulltext.disabled` and `nuxeo.vcs.fulltext.search.disabled`. So it would be better to inverse the logic and use an `enabled` prop, as we have tons of those.

Reply:
I took example on what was done for the WorkManager with `nuxeo.work.processing.disable` both options are going to be used together, it will be weird to have:
nuxeo.work.processing.disable=true
nuxeo.stream.processing.enabled=false
wdyt?

Reply:
ok, I am going to add a  `nuxeo.work.processing.enabled` in the WM (keeping compat with old option)